gradio/blocks.py

```diff
@@ -1637,11 +1637,11 @@ def queue(
             with gr.Blocks() as demo:
                 button = gr.Button(label="Generate Image")
                 button.click(fn=image_generator, inputs=gr.Textbox(), outputs=gr.Image())
-            demo.queue(concurrency_count=3)
+            demo.queue(concurrency_count=1)
             demo.launch()
         Example: (Interface)
             demo = gr.Interface(image_generator, gr.Textbox(), gr.Image())
-            demo.queue(concurrency_count=3)
+            demo.queue(concurrency_count=1)
```

Comment:
Similarly, you can remove here. 
```suggestion
            demo.queue(max_size=20)
```